ml-proto/README.md

```diff
@@ -126,10 +126,10 @@ type expr =
   | GetParam of var                         (* read parameter
   | GetLocal of var                         (* read local variable
   | SetLocal of var * expr                  (* write local variable
-  | LoadGlobal of var                       (* read global variable
-  | StoreGlobal of var * expr               (* write global variable
-  | Load of loadop * expr                   (* read memory address
-  | Store of storeop * expr * expr          (* write memory address
+  | Load of memop * expr                    (* read memory address
+  | Store of memop * expr * expr            (* write memory address
+  | LoadExtend of extendop * expr           (* read memory address with sign- or zero-extension
+  | StoreTrunc of truncop * expr * expr     (* write memory address with truncation
```

Comment:
Here and elsewhere in this patch, the design repo calls this "wrap" instead of "trunc", for consistency with the conversion opcodes.